tests/e2e/specs/amp/admin-bar.test.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * WordPress dependencies
+ */
+import { deactivatePlugin, activatePlugin, createURL } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal depedencies
+ */
+import {
+	activateAMPWithMode,
+	deactivateUtilityPlugins,
+	setupSiteKit,
+	useRequestInterception,
+} from '../../utils';
+import * as adminBarMockResponses from '../modules/analytics/fixtures/admin-bar';
+
+let mockBatchResponse;
+
+describe( 'AMP Admin Bar compatibility', () => {
+	beforeAll( async () => {
+		await setupSiteKit();
+		await activatePlugin( 'e2e-tests-admin-bar-visibility' );
+		await activateAMPWithMode( 'standard' );
+
+		await page.setRequestInterception( true );
+		useRequestInterception( ( request ) => {
+			if ( request.url().match( 'google-site-kit/v1/data/' ) ) {
+				request.respond( {
+					status: 200,
+					body: JSON.stringify( mockBatchResponse ),
+				} );
+			} else {
+				request.continue();
+			}
+		} );
+	} );
+
+	beforeEach( async () => {
+		mockBatchResponse = [];
+		await page.goto( createURL( '/hello-world' ), { waitUntil: 'load' } );
+	} );
+
+	afterAll( async () => {
+		await deactivatePlugin( 'amp' );
+		await deactivateUtilityPlugins();
+	} );
+
+	it( 'Site Kit admin bar component display', async () => {
```

Comment:
I think this may have been copied from another test but this could use a better name.

Something like "it has a functional Site Kit admin bar menu in AMP" would be a bit more descriptive.

Reply:
Updated